examples/control-elements/media-loading-indicator.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width">
+  <script type="module" src="../../dist/media-loading-indicator.js"></script>
+  <style>
+    media-loading-indicator {
+      --media-icon-color: #f0f;
+    }
+  </style>
+</head>
+<body>
+
+<h1>&lt;media-loading-indicator&gt;</h1>
+
+<h3>Default (not loading)</h3>
```

Comment:
nit (non-blocking): since this is the only case where our component ends up being auto-hidden based on attributes, maybe add mention of it in the `<h3>` description